src/index.css

```diff
@@ -195,3 +213,64 @@ hr {
     padding: 0 1.5em;
   }
 }
+
+/* Accordion */
```

Comment:
can we move all these to a separate accordian css file? index is just meant for things that are truly global